mathics/core/characters.py

```diff
@@ -23,20 +23,20 @@ def unicode_equivalent(k: str, v: dict):
 # Conversion from WL to the fully qualified names
 WL_TO_PLAIN_DICT = {re.escape(v["wl-unicode"]): f"\\[{k}]" 
```

Comment:
This is the kind of thing that can be computed once per install and put into a JSON table.

Reply:
> This is the kind of thing that can be computed once per install and put into a JSON table. And re_from keys then would work on a set or a list rather than a dictionary.

Yep, I plan to move this stuff to a install-time script after I get the tests to pass

Reply:
Essentially, this is just a draft of the install-time script 😁️

Reply:
@rocky Alright, it looks like the tests are passing. Could you help me figure out where/how should I place the install script?

Reply:
Sure - but I can't do this right now. 

Depending on how my day job coding goes, in a day or so I'll copy all of this to a new repostory and set that up. Thanks.

Reply:
I've said it before but I'll say it again. Overall I think this is great work! @GarkGarcia 

It addresses one of the dark area of WL that we can bring light to. I think when this and the parser are put in a separate module there will be lots of uses of this outside of Mathics. I am pretty sure there are no other good and maintainable alternatives. 

After this is

Reply:
> Sure - but I can't do this right now.
> 
> Depending on how my day job coding goes, in a day or so I'll copy all of this to a new repostory and set that up. Thanks.

Great! I've set up the code in a way to makes it easy for you to extract it to a separate package: just copy and paste mathics/core/characters.py and transfer the section marked as `INITIALIZATION` at the beginning to the installation script.

> I've said it before but I'll say it again. Overall I think this is great work! @GarkGarcia 

Thanks!

> It addresses one of the dark area of WL that we can bring light to. I think when this and the parser are put in a separate module there will be lots of uses of this outside of Mathics. 

Yes, I can already see us using this module to generate the tables in the developer docs instead of the CSV files (the YAML file in here has all of the information they have + some other stuff)